etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/SendMessageHelper.java

```diff
@@ -21,67 +21,67 @@ public static SendMessageHelper getInstance() {
 
     private SendMessageHelper() {}
 
-    public void savePartnerMetadataForReceivedMessage(PartnerMetadata partnerMetadata) {
-        if (partnerMetadata.receivedSubmissionId() == null) {
+    public void savePartnerMetadataForOutboundMessage(PartnerMetadata partnerMetadata) {
+        if (partnerMetadata.outboundMessageId() == null) {
             logger.logWarning(
-                    "Received submissionId is null so not saving metadata for received message");
+                    "Outbound messageId is null so not saving metadata for outbound message");
             return;
         }
         try {
-            partnerMetadataOrchestrator.updateMetadataForReceivedMessage(partnerMetadata);
+            partnerMetadataOrchestrator.updateMetadataForOutboundMessage(partnerMetadata);
         } catch (PartnerMetadataException e) {
             logger.logError(
-                    "Unable to save metadata for receivedSubmissionId "
-                            + partnerMetadata.receivedSubmissionId(),
+                    "Unable to save metadata for outboundMessageId "
+                            + partnerMetadata.outboundMessageId(),
                     e);
         }
     }
 
-    public void saveSentMessageSubmissionId(String receivedSubmissionId, String sentSubmissionId) {
-        if (sentSubmissionId == null || receivedSubmissionId == null) {
+    public void saveInboundMessageId(String outboundMessageId, String inboundMessageId) {
```

Comment:
This seems to do both inbound and outbound message saving. @basiliskus Should I leave the old name if that's the case?

Reply:
Hmmm I think we should remove the `Sent` part and make it plural. So it would be renamed as `saveMessageSubmissionIds` or `saveSubmissionIds`

Reply:
I'll rename it to `saveMessageSubmissionIds`